deployment.yaml.erb

```diff
@@ -342,11 +379,14 @@ spec:
 <%- if @Eth_Etherbase && @Eth_MinerThreads -%>
            --mine \
 <% end -%>
+<%- if @metrics -%>
+           --metrics \
```

Comment:
i had a look, this for some reason can't be configured through the geth config file, so this is correct :+1: